diff --git a/yin_yang/daemon_handler.py b/yin_yang/daemon_handler.py
index 6b845bf..8cb9f5a 100644
--- a/yin_yang/daemon_handler.py
+++ b/yin_yang/daemon_handler.py
@@ -3,6 +3,8 @@ import shutil
 import subprocess
 from enum import Enum, auto
 from pathlib import Path
+import os
+import stat
 
 from .config import ConfigWatcher, config
 from .meta import ConfigEvent, Modes
@@ -19,8 +21,12 @@ def create_files():
         SYSTEMD_PATH.mkdir(parents=True, exist_ok=True)
     if not TIMER_PATH.is_file():
         shutil.copy('./resources/yin_yang.timer', TIMER_PATH)
+        st = os.stat(TIMER_PATH)
+        os.chmod(TIMER_PATH, st.st_mode | stat.S_IWRITE)
     if not SERVICE_PATH.is_file():
         shutil.copy('./resources/yin_yang.service', SERVICE_PATH)
+        st = os.stat(SERVICE_PATH)
+        os.chmod(SERVICE_PATH, st.st_mode | stat.S_IWRITE)
 
 
 def run_command(command, **kwargs):
